src/uwtools/tests/test_cli.py

```diff
@@ -195,15 +195,20 @@ def test__dispatch_config(params):
     vars(args).update({STR.submode: submode})
     with patch.object(cli, funcname) as func:
         cli._dispatch_config(args)
-    assert func.called_once_with(args)
+    func.assert_called_once_with(args)
 
 
 def test__dispatch_config_compare():
     args = ns()
     vars(args).update({STR.file1path: 1, STR.file1fmt: 2, STR.file2path: 3, STR.file2fmt: 4})
     with patch.object(cli.uwtools.config.core, "compare_configs") as compare_configs:
         cli._dispatch_config_compare(args)
-    assert compare_configs.called_once_with(args)
+    compare_configs.assert_called_once_with(
+        config_a_path=args.file_1_path,
+        config_a_format=args.file_1_format,
+        config_b_path=args.file_2_path,
+        config_b_format=args.file_2_format,
+    )
```

Comment:
Fixed several instances of improper assertions:
1. Replaced `assert` statements with invocations of mock objects' `asssert_called_once_with()` methods.
2. With the previous change in place, tests (correctly) started failing, so updated expected call arguments with the correct values.

This pattern repeats throughout the remainder of this module.